WeScan/Edit/EditScanViewController.swift

```diff
@@ -151,11 +156,33 @@ final class EditScanViewController: UIViewController {
             uiImage = UIImage(ciImage: filteredImage, scale: 1.0, orientation: .up)
         }
         
-        let results = ImageScannerResults(originalImage: image, scannedImage: uiImage, detectedRectangle: scaledQuad)
+        let finalImage = fixImageRotation(image: uiImage)
+        let results = ImageScannerResults(originalImage: image, scannedImage: finalImage, detectedRectangle: scaledQuad)
         let reviewViewController = ReviewViewController(results: results)
         
         navigationController?.pushViewController(reviewViewController, animated: true)
     }
+    
+    private func fixImageRotation(image: UIImage) -> UIImage {
```

Comment:
Can you add documentation on what this method does?

Reply:
@justJS will be very helpful to double check my tests.